src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -196,6 +196,11 @@
       <arg name="text" type="s" direction="out"/>
     </signal>
 
+    <!-- Straight from your door to our console, unadulterated! -->
+    <signal name="RawMessage">
```

Comment:
> <RawMessage>Frobnicating the baz...</><RawMessage>done!\n</> that might be simpler with something like <BeginTask>Frobnicating the baz...</><EndTask/> ?

The issue with this is that we want it to work in code that could both be run inside and outside the daemon. So we would probably have to add another layer of abstraction which can determine whether to emit a message (and somehow retrieve the associated Transaction without plumbing it throughout the code) or to just print on stdout.